assets/js/app/inventory/ItemModificationModal.jsx

```diff
@@ -4,7 +4,7 @@ import { FormGroup, Button, Modal, FormControl, ControlLabel }  from 'react-boot
 import { getCookie } from '../../csrf/DjangoCSRFToken'
```

Comment:
I'm trusting you on front end stuff but I should review it for my understanding later